src/rpc/RPCHelpers.cpp

```diff
@@ -663,6 +663,10 @@ traverseOwnedNodes(
     boost::asio::yield_context& yield,
     std::function<void(ripple::SLE&&)> atOwnedNode)
 {
+    if (!backend.fetchLedgerObject(
+            ripple::keylet::account(accountID).key, sequence, yield))
+        return Status{Error::rpcACT_NOT_FOUND};
```

Comment:
@cjcobb23 since this is part of Clio, do we not have a hardcoded error string in addition to the rpc error code (rippled ErrorCodes.h)?

Reply:
the string is added by default, based on the code